data-raw/qc-markdowns/adult-holding-redd-and-carcass-surveys/feather-river/feather-carcass-survey-qc.Rmd

```diff
@@ -0,0 +1,539 @@
+---
+title: "Feather Carcass QC "
+author: "Inigo Peng"
+date: '2022-08-18'
+output: rmarkdown::github_document
+---
+  
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, warning = FALSE, fig.width=15, fig.height=10)
+
+library(tidyverse)
+library(lubridate)
+library(googleCloudStorageR)
+library(ggplot2)
+library(scales)
+library(readxl)
+library(janitor)
+library(hms) #?as_hms()
+library(RODBC)
+library(knitr)
+library(wesanderson)
+```
+
+
+# Feather River Carcass Data
+
+## Description of Monitoring Data
+
+**Timeframe:** 2000 to 2016
+  
+**Video Season:** 
+  
+**Completeness of Record throughout timeframe:** 
+
+* Data collected varies between different seasons. Cwt has the highest variability. Stop collecting "Chops Recovery" after (2011?); earlier years have more detailed chops data and had separate table for tags data; later chops data and tags data combines into one.
+  
```

Comment:
We should try to figure out the reaches (like how Ryan gave us a map). Anything like that included in Feather Escapement protocol? Otherwise we can ask Kassie.